src/components/data/LocalVariablesContent.js

```diff
@@ -18,13 +18,31 @@ import * as StateHistory from 'src/libs/state-history'
 import * as Utils from 'src/libs/utils'
 
 
+export const getDisplayedAttribute = arr => {
+  const { key, value, description } = _.assign(...arr, {})
+  return [key, value, description ?? '']
+}
+
+const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
+const isDescriptionKey = k => k.endsWith(DESCRIPTION_SUFFIX)
```

Comment:
you could let the auto-currying do the work for you:
```suggestion
const isDescriptionKey = _.endsWith(DESCRIPTION_SUFFIX)
```

Reply:
Done.